src/game-board/game-board.html

```diff
@@ -60,15 +60,24 @@
         //1 to avoid an out of bounds exception.
         var stackCardRank = rankList.indexOf(stack[stack.length-1].rank);
         //Simple comparison here. If the the card we want to place is a rank lower than the card on top of the stack (suit doesn't matter here)
-        //Then we return true. Else we return false. 
-        if(cardRank == stackCardRank - 1){
-          return true;
-        } else{
-          return false;
+        //Then we return true. Else we return false.
+        if(isFoundation == true){
```

Comment:
Mb, I thought I had correct indentation in Atom, but I think it translated weirdly over here. In any case I think it is ready to merge now that I added additional logic to account for identical suits in the foundation stacks